src/sdl2_compat.c

```diff
@@ -1111,7 +1111,7 @@ SDL_GetRevisionNumber(void)
 }
 
 
-DECLSPEC void SDLCALL
+DECLSPEC int SDLCALL
```

Comment:
This, and the corresponding change to SDL_OpenAudio below, can go in directly without this PR. (How did we miss this?..)

Reply:
yes, don't know :) at first I though this was intended but let's fix it within this pr anyway

Reply:
> (How did we miss this?..)

Likely it was in the initial framework I copy/pasted from sdl12-compat, where this didn't return an int.